src/main/java/com/microsoft/azure/NetworkAclSample.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.azure;
```

Comment:
Should we update the package as well? All Track 2 libraries use `com.azure` instead of `com.microsoft.azure`, which Track 1 used.
```suggestion
package com.azure;
```

Reply:
Fixed in the new version.